File: C_Sharp_code/Example/WindowsFormsApp/VB_demo21/Car.vb
'Public NotInheritable Class Car '加上NotInheritable關鍵字 密封不能被繼承
Public Class car
    Private _name As String
    Public Property Name As String
        Get
            Return _name
        End Get
        Set(value As String)
            Me._name = value
        End Set
    End Property

    Public Sub New()
        Me.New("Is Car")
    End Sub
    Public Sub New(ByVal argName As String)
        Me._name = argName
    End Sub

    Public Overridable Sub Display()
        Console.WriteLine("Car method")
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo20/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CS_demo20
{
    //sealed class Car //加上sealed關鍵字 密封不能被繼承
     class Car
    {
        private string _name;
        public string Name
        {
            get
            {
                return _name;
            }
            set
            {
            _name = value;
            }
        }
        public Car():this("Is Car")
        {

        }
        public Car(string argName)
        {
            this._name = argName;
        }
        public virtual void display()
        {
            Console.WriteLine("Car method");
        }
    }
}
